src/v/pandaproxy/rest/handlers.cc

```diff
@@ -285,13 +285,12 @@ create_consumer(server::request_t rq, server::reply_t rp) {
 
     auto base_uri = make_consumer_uri_base(rq, group_id);
     auto group_shard{consumer_shard(group_id)};
```

Comment:
I believe all of the `auto group_shard{consumer_shard(group_id)};` are dead code now.